ext/dapr-ext-workflow/dapr/ext/workflow/util.py

```diff
@@ -17,6 +17,7 @@
 
 from dapr.conf import settings
 
+DAPR_API_TOKEN_HEADER = 'dapr-api-token'
```

Comment:
Please import this instead so we can change everything in one place if it ever must change.

`from dapr.ext.workflow.util import DAPR_API_TOKEN_HEADER`

Reply:
yeah, used from original place only now.